xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -1097,15 +1097,41 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Range when fully fuelled or charged. +v2.0</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="FullCharge" type="WattHoursType" minOccurs="0">
+			<xsd:element name="FullCharge" type="WattHoursType" minOccurs="0"/>
+			<xsd:sequence>
+				<xsd:element name="equipmentProfiles" type="vehicleEquipmentProfileRefs_RelStructure" minOccurs="0">
+					<xsd:annotation>
+						<xsd:documentation>Equipment profiles assoicated with model. +v1.2.2</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+			<xsd:element ref="VehicleModelProfileRef" minOccurs="0"/>				
+			</xsd:sequence>			
+			<xsd:element name="CustomerServiceContactDetails" type="ContactStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Especially very large vehicles and also some ALTERNATIVE MODE OF OPERATION offers need the possibility to contact the VEHICLE directly.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<xsd:group name="VehicleCodeGroup">
+		<xsd:annotation>
+			<xsd:documentation>Alternative code elements for VEHICLE.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="OperationalNumber" type="xsd:normalizedString" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Operational number of VEHICLE.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element ref="PrivateCode" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Energy required for a full charge. +v2.0</xsd:documentation>
+					<xsd:documentation>Private code of the vehicle.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
 			<xsd:sequence>
 				<xsd:element name="equipmentProfiles" type="vehicleEquipmentProfileRefs_RelStructure" minOccurs="0">
```

Comment:
Isn't that perhaps superfluous? We already have the equipment profiles for the model by *equipmentProfiles* in the *VehicleModelGroup* (and for the specific vehicle we have *actualVehicleEquipments*).